src/main/java/org/matsim/mosaik2/prepare/GetCountData.java

```diff
@@ -0,0 +1,144 @@
+package org.matsim.mosaik2.prepare;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.log4j.Logger;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class GetCountData {
+
+	private static final Logger logger = Logger.getLogger(CreateCounts.class);
+
+	Map<String, CountingData> countData(String filePath1, String filePath2, Map<String, NodeMatcher.MatchedLinkID> nodeMatcher) throws IOException {
+
+		var result1 = readData(filePath1, nodeMatcher);
+		var result2 = readData(filePath2, nodeMatcher);
+
+		for (var entry : result1.entrySet()) {
+
+			result2.put(entry.getKey(), entry.getValue());
+
+		}
+
+		logger.info("###############################################");
+		logger.info("#\t\t\t\t\t\t\t\t\t\t\t\t#");
+		logger.info("#\t\t\t All Counts were imported! \t\t\t#");
+		logger.info("#\t\t\t  " + result2.keySet().size() + " stations were found!\t\t\t#");
+		logger.info("#\t\t\t\t\t\t\t\t\t\t\t\t#");
+		logger.info("###############################################");
+		return result2;
+
```

Comment:
I don't think whitespace between closing brackets and end of code is necessary. Please also remove at other places